gapic-generator-java/src/main/java/com/google/api/generator/engine/ast/PackageInfoDefinition.java

```diff
@@ -22,8 +22,14 @@
 
 @AutoValue
 public abstract class PackageInfoDefinition implements AstNode {
+  public static final PackageInfoDefinition EMPTY = builder().setPakkage("").build();
+
   public abstract String pakkage();
 
+  public boolean shouldGenerateFile() {
+    return pakkage().isEmpty();
+  }
```

Comment:
Should this be `!pakkage().isEmpty()`? If so, this only won't generate only pakkage is set to `""`? 

Do we need to also check fileHeaders, headerCommentStatements, and annotations?

Reply:
That's right. However we decided to have this PR being less invasive regarding package info, so we removed this change.